ios/Runner/Info.plist

```diff
@@ -35,6 +33,10 @@
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSCameraUsageDescription</key>
+	<string>Can I use the camera please?</string>
+	<key>NSMicrophoneUsageDescription</key>
+    <string>Can I use the mic please?</string>
```

Comment:
Can't figure out what these strings are for. Are they displayed in the app? We might need to update them with a better content

Reply:
No theses strings are for iOS configuration, basically these strings show up when we request the user permission to use his cam or mic